docs/metrics/v3.md

```diff
@@ -377,3 +385,13 @@ The standard metrics group for GoCollector is not shown below.
 | `minio_scanner_last_activity_seconds`      | `gauge`   | Time elapsed (in seconds) since last scan activity         | `server` |
 | `minio_scanner_objects_scanned`            | `counter` | Total number of unique objects scanned since server start  | `server` |
 | `minio_scanner_versions_scanned`           | `counter` | Total number of object versions scanned since server start | `server` |
+
+### `/ilm`
+
+| Name                                                  | Type      | Help                                                                       | Labels   |
+|-------------------------------------------------------|-----------|----------------------------------------------------------------------------|----------|
+| `minio_cluster_ilm_expiry_pending_tasks`              | `counter` | Number of pending ILM expiry tasks in the queue                            | `server` |
+| `minio_cluster_ilm_transition_active_tasks`           | `counter` | Number of active ILM transition tasks                                      | `server` |
```

Comment:
same as above for `minio_cluster_ilm_transition_active_tasks`

Reply:
done